src/store/index.js

```diff
@@ -25,12 +25,14 @@ const updateGraphOnChange = (store) => {
  * @param {object} state Redux state snapshot
  */
 const saveStateToLocalStorage = (state) => {
+  //Parameters do not get saved to local storage
+  const { parameters, ...nodeTypeDisabled } = state.nodeType.disabled;
```

Comment:
It looks like the `parameters` variable being pulled here isn't actually in use, so I think should be removed if that's right? Not sure what the comment here refers to as well, could you clarify it a bit more?

Reply:
Hi Liam, 

So basically, I am de-structuring nodeType.disabled and passing all the values except the parameters state to localStorage. This is so that parameters are always turned off for the user (even when they turn it on and refresh their page)

Reply:
I updated the comment - //The state of parameters when enabling/disabling does not get saved to local storage

Reply:
Oh looks like I didn't catch the `...` usage here in this destructuring, that makes more sense now, so I think it's fine.